src/Option/README.md

```diff
@@ -0,0 +1,24 @@
+# Option:
+
+Provides an Option component that is passed as a child to the ListBox component.
+
+## API
+
+### `aria-selected` (boolean; optional):
+The ListBox component will handle the `aria-selected` attribute itself, but the `aria-selected` prop can be used as an override, if necessary.
+
+### `children` (element array; required)
+`children` specifies the list of elements that will be displayed within the Option component.
+
+### `className` (string array; optional)
+`className` specifies Bootstrap class names to apply to the Option component. The default is an empty array. (do I mention the existing classnames here?)
+
+### `tag` (string; optional)
+`tag` is used to specify the element type of the rendered Option component. The default is div. Example alternative include li, Button, etc.
+
+### `onOptionSelect` (function, optional)
+`onOptionSelect` is a function that is called when an Option component is selected, by mouse or by keyboard, to perform some additional action.
+
+### `other`
+`other` specifies any additional attributes on the option element, which can include aria-labelledby, id, key, etc.
```

Comment:
The Button component does something similar, and I like it, but I am a little concerned this will give the impression that the prop is called "other" instead of a catch-all for all other props sent into the component. Does anyone have a better way to get the meaning across?